pkgs/tools/misc/brltty/default.nix

```diff
@@ -46,7 +49,10 @@ stdenv.mkDerivation rec {
     "--with-writable-directory=/run/brltty"
     "--with-updatable-directory=/var/lib/brltty"
     "--with-api-socket-path=/var/lib/BrlAPI"
-  ];
+  ] ++ lib.optional espeak-ngSupport [
+      "--with-espeak_ng=$espeak-ng"
```

Comment:
What is `$espeak-ng` supposed to achieve? Shell variables cannot contain dashes, and I assume autofoo/gnumake, which will interpolate it, has similar same rule. It’s not like there is `$espeak` shell variable.